index.js

```diff
@@ -340,6 +347,14 @@ class HyperDHT extends DHT {
       value
     }, from)
   }
+
+  _availableStreamId () {
+    while (true) {
+      const id = (Math.random() * 0xffffffff) >>> 0
```

Comment:
hate to be that guy, but this should be `random() * 0x100000000`

Reply:
I feel like this should be a public method on the pool also instead now

Reply:
Always off by 1 🙈